static/informed_consent.html

```diff
@@ -1,419 +1,413 @@
+
+          <p class="list-group-item-text">
+          You will not receive compensation for participation in this study.
+          </p>
+
+        </div>
+
+        <div class="list-group-item">
+          <h4 class="list-group-item-heading termsHeading">CONTACT INFORMATION</h4>
+
+          <p>
+          If you have any questions regarding the study, please contact Principal Investigators Dr. Ronald
+          Deibert or Dr. Phillipa Gill. If you have questions regarding your rights as a research participant
+          you can contact the University of Toronto Office of Research Ethics at <br/>
+          ethics.review@utoronto.ca or 416-946-3273
+          </p>
+          <p>
+          Dr. Ronald J. Deibert (Principal Investigator) <br/>
+          Munk School of Global Affairs <br/>
+          University of Toronto <br/>
+          1 Devonshire Place, Toronto, Ontario, M5S 3K7 Canada <br/>
+          Tel: (416) 946-8916 Fax: (416) 946-8915 <br/>
+          Email: r.deibert@utoronto.ca <br/> <br/>
+          </p>
+          <p>
+          Dr. Phillipa Gill (Principal Investigator) <br/>
+          Stony Brook University <br/>
+          Computer Science Building, Stony Brook University, Stony Brook, NY 11794-4400 <br/>
+          Tel: (631) 632-8470 <br/>
+          Email: phillipa@cs.stonybrook.edu
+          </p>
+        </div>
+
+        <div class="list-group-item">
+
+          <h4 class="list-group-item-heading termsHeading">CONSENT FORM</h4>
+          <p>
+          Study Title: Internet Censorship Lab (ICLab)
+          </p>
+          <p>
+          My verbal consent or signature on this consent form means the following:
+          </p>
+          <p>
+          The study has been fully explained to me and all of my questions have been answered to my
+          satisfaction
+          <p>
+
+          <div class="list-group">
+            <div class="list-group-item">
+              <p>I understand the requirements and the risks of the study,</p>
+            </div>
+
+            <div class="list-group-item">
+              <p>I understand that the research results from this study will be shared publicly in
+              publications and presentations,</p>
+            </div>
+
+            <div class="list-group-item">
+              <p>I understand that my personally identifiable information (i.e. name, IP address) will not
+              be presented publicly and will only be accessible by the research team.</p>
+            </div>
+
+            <div class="list-group-item">
+              <p>I confirm that I have the administrative authority to enroll my organization into this
+              study,</p>
+            </div>
+
+            <div class="list-group-item">
+              <p>I agree to take part in this study, and</p>
+            </div>
+
+            <div class="list-group-item">
+              <p>I will be given I can be provided with a copy of this signed and dated consent form.</p>
```

Comment:
I think this is a typo. Was the typo part of the accepted protocol and can we change it?